bcipy/helpers/tests/test_stimuli.py

```diff
@@ -314,5 +359,30 @@ def test_image_fixation_uses_default(self):
         self.assertEqual(expected, response)
 
 
+class TestAlphabetize(unittest.TestCase):
+
+    def setUp(self) -> None:
+        self.list_to_alphabetize = ['Z', 'Q', 'A', 'G']
+
+    def test_alphabetize(self):
+
+        expected = ['A', 'G', 'Q', 'Z']
+        response = alphabetize(self.list_to_alphabetize)
+        self.assertEqual(expected, response)
+
+    def test_alphabetize_image_name(self):
+        list_of_images = ['testing.png', '_ddtt.jpeg', 'bci_image.png']
+        expected = ['bci_image.png', 'testing.png', '_ddtt.jpeg']
+        response = alphabetize(list_of_images)
+        self.assertEqual(expected, response)
+
+    def test_alphabetize_special_characters_at_end(self):
+        character = '<'
+        expected = ['A', 'G', 'Q', 'Z', character]
+        self.list_to_alphabetize.append(character)
```

Comment:
It might be more instructive to put the backspace character in the middle or near the front, since we expect it to be sorted near the end. I would also add the space character here. I tested locally and these work as expected. 👍